### ISSUE TITLE
Multiple userDataDirs per bot?

### ISSUE THREAD

**Issue by iwishiknewcoding** [gabrielbrop/haxball-server#11]
Hello @gabrielbrop, I currently have several hosts with their own `userDataDir`-s. Can I somehow provide correct dirs to each of the bots? From the current implementation, I think we can only pass a single dir that will be shared across bots, is that correct?

**Comment by gabrielbrop** [comment]
Yes, you can only pass a single dir and it'll be shared by all rooms. By the way, I recommend using some real database like MySQL or MongoDB instead of localStorage.
